rpcserver.go

```diff
@@ -4429,15 +4429,17 @@ func (r *rpcServer) UpdateChannelPolicy(ctx context.Context,
 		return nil, fmt.Errorf("unknown scope: %v", scope)
 	}
 
+	switch {
 	// As a sanity check, we'll ensure that the passed fee rate is below
 	// 1e-6, or the lowest allowed fee rate, and that the passed timelock
 	// is large enough.
-	if req.FeeRate < minFeeRate {
+	case req.FeeRate != 0 && req.FeeRate < minFeeRate:
```

Comment:
comment looks out of date?

Reply:
kek should have been more clear that it also says "we'll ensure that the passed fee rate is below 1e-6", instead of above the minimum.